src/combinatorics/simple_graph/regularity/energy.lean

```diff
@@ -44,4 +45,8 @@ div_le_of_nonneg_of_le_mul (sq_nonneg _) zero_le_one $
     ... = P.parts.off_diag.card : nat.smul_one_eq_coe _
     ... ≤ _ : by { rw [off_diag_card, one_mul, ←nat.cast_pow, nat.cast_le, sq], exact tsub_le_self }
 
+@[simp, norm_cast] lemma coe_energy {𝕜 : Type*} [linear_ordered_field 𝕜] :
+  (P.energy G : 𝕜) = (∑ uv in P.parts.off_diag, G.edge_density uv.1 uv.2 ^ 2) / P.parts.card ^ 2 :=
+by { rw energy, norm_cast }
```

Comment:
It's not clear to me why this is a good lemma; why do you want to unfold `energy` in every linear_ordered_field other than `ℚ`?

Reply:
`energy` is internal to SRL so API concerns don't really apply. This is just to shorten the later proof

Reply:
Can't you just use `rw energy, norm_cast` inside the later proof? Doing so also removes a forward-porting obligation.

Reply:
How? `combinatorics.simple_graph.energy` isn't forward-ported

Reply:
Yes it is...

Reply:
Oh... You mean this file is _already_ out of sync?

Reply:
Hmm, no I genuinely thought it wasn't ported yet, but indeed it looks like it is. At any rate, I see no reason to drop the lemma. It's useful in the proof and its purpose is the proof.